src/style/sections/exports/exports.less

```diff
@@ -165,6 +165,33 @@
     flex-direction: row;
 }
 
+.layer-exports__header {
+   width: 100%;
+
+   .formline.formline__space-between {
```

Comment:
Okay, in main.less:157 and in drop-down.less:61, I added .export to the selector. That adds the underlines to the inputs. Then, you should only need to keep the font-size property that you have in this block